webapp/messages/en.json

```diff
@@ -60,20 +60,24 @@
     }
   },
   "demos": {
-    "heading": "Demos",
-    "sub-heading": "Learn and explore with Hemi developer demos",
-    "hemihatchlings": {
-      "heading": "Hemi Hatchlings",
+    "bitcoin-kit": {
+      "heading": "Hemi Bitcoin Kit Demo",
       "sub-heading": "Learn how Ethereum & Bitcoin finality works"
```

Comment:
Should we use the same sub-heading as we have used for hemihatchlings?

Reply:
It's the one from the design, but you're correct, it doesn't sound like it's the one. Maybe @rmilrad  could provide the appropriate texts once he's back

Reply:
Wording updated here [9193980](https://github.com/hemilabs/ui-monorepo/pull/722/commits/9193980ea9c77797be65747adfbe5547bdd6dee7)